configure.ac

```diff
@@ -1254,40 +1254,12 @@ dnl ncurses takes precedence.
 dnl
 AC_CHECK_HEADERS([curses.h curses/curses.h ncurses.h ncurses/ncurses.h])
 
-dnl Although n-library is preferred, only look for it if the n-header was found.
-CURSESLIBS=''
-PC_NAME_CURSES=""
-PC_LIBS_CURSES=""
-AS_IF([test x"$ac_cv_header_ncurses_h" = x"yes"], [
-    AC_CHECK_LIB(ncurses, printw,
-                 [AC_DEFINE(HAVE_LIBNCURSES,1, have ncurses library)])
-    CURSESLIBS=`$PKG_CONFIG --libs ncurses` || CURSESLIBS='-lncurses'
-    PC_NAME_CURSES="ncurses"
-])
-
-AS_IF([test x"$ac_cv_header_ncurses_ncurses_h" = x"yes"], [
-    AC_CHECK_LIB(ncurses, printw,
-                 [AC_DEFINE(HAVE_LIBNCURSES,1, have ncurses library)])
-    CURSESLIBS=`$PKG_CONFIG --libs ncurses` || CURSESLIBS='-lncurses'
-    PC_NAME_CURSES="ncurses"
-])
-
-dnl Only look for non-n-library if there was no n-library.
-AS_IF([test x"$CURSESLIBS" = x"" && test x"$ac_cv_header_curses_h" = x"yes"], [
-    AC_CHECK_LIB(curses, printw,
-                 [CURSESLIBS='-lcurses'; AC_DEFINE(HAVE_LIBCURSES,1, have curses library)])
-    PC_LIBS_CURSES="$CURSESLIBS"
-])
-
-dnl Only look for non-n-library if there was no n-library.
-AS_IF([test x"$CURSESLIBS" = x"" && test x"$ac_cv_header_curses_curses_h" = x"yes"], [
-    AC_CHECK_LIB(curses, printw,
-                 [CURSESLIBS='-lcurses'; AC_DEFINE(HAVE_LIBCURSES,1, have curses library)])
-    PC_LIBS_CURSES="$CURSESLIBS"
-])
-
-AS_IF([test x"$CURSESLIBS" != x""],
-      [PCMK_FEATURES="$PCMK_FEATURES ncurses"])
+save_LIBS=$LIBS
```

Comment:
Should we clear LIBS here so that CURSESLIBS doesn't duplicate everything that was already in it? The only question is whether AC_SEARCH_LIBS could possibly need something from the existing LIBS. OTOH listing a library twice probably doesn't cause any significant issues.